articles/api/authentication/_change-password.md

```diff
@@ -79,6 +79,10 @@ This endpoint only works for database connections.
 - If a password is provided, when the user clicks on the confirm password change link, the new password specified in this POST will be set for this user.
 - If a password is NOT provided, when the user clicks on the password change link they will be redirected to a page asking them for a new password.
 - The sample auth0.js script uses the library version 8. If you are using auth0.js version 7, please see this [reference guide](/libraries/auth0js/v7).
+- This endpoint will return three HTTP Response Headers, that provide relevant data on its [rate limits](/policies/rate-limits):
+  - `X-RateLimit-Limit`: Number of requests allowed for a period of time.
+  - `X-RateLimit-Remaining`: Number of requests available within that period of time. Each new request reduces this number by 1.
```

Comment:
@angelaLazar I would like to have more details on how this works. I've seen that the `X-RateLimit-Remaining` does not go from 0 to 10 when the `X-RateLimit-Reset` is reached, instead it is increased gradually.